app/backend/requirements.in

```diff
@@ -1,14 +1,15 @@
 azure-identity
 quart
 quart-cors
-openai[datalib]==0.28.1
+openai[datalib]>=1.3.5,<2.0.0
```

Comment:
Do we need the upper constraint?

Reply:
the major version change indicates some sort of breaking change, so I'd rather have the constraint there just in case.

Reply:
We explicitly have a pip-compile step now, so I'm wondering why an upper constraint would be needed in the .in file, and why we'd have it for openai and not for other packages?